dapp/views/transactions/transactions_list/transactions_list.css

```diff
@@ -1,3 +1,43 @@
 .transactions-list {
-  height: calc(100% - 60px);
+  height: calc(100% - 40px);
+  overflow-y: overlay;
+
+  &::-webkit-scrollbar {
```

Comment:
Inspired by this comment, I have reworked scrollbars to use https://github.com/pascalduez/postcss-scrollbar. The scrollbar is now compatible across browsers - it is good, however some disadvantages:
- Not as thin as the design, but `postcss-scrollbar` does not support thinner.
- Can't use `var`s to set colours.
- Only indirectly related to scrollbars, but on Firefox the reason why the buttons were pushed out of the bottom is because the height `calc` doesn't work. I cannot find a way to fix this except by explicitly setting `height: 312px`

Reply:
That may be some incompatibilities in the css box model. We can go back to this in the next iteration. Fixed height is fine for me here as long as subviews remain flexible.